### ISSUE TITLE
Reactions not available in the Moodle Mobile application

### ISSUE THREAD

**Issue by tallthom** [ponlawat-w/moodle-local_reactforum#5]
If possible, it would be good to allow reacting within the Moodle Mobile application. I realize this functionality is not enabled by default, but this is an enhancement request to improve reactions moving forward.

**Comment by rschrenk** [comment]
Yes, this would indeed increase the benefit from this plugin, as the forum is very often used with the mobile app.
